src/Spryker/Client/PriceProductStorage/Storage/PriceProductMapper.php

```diff
@@ -93,8 +93,15 @@ protected function getPriceProductTransfersFromPriceData(
      */
     protected function setPriceData(PriceProductTransfer $priceProductTransfer, array $prices): PriceProductTransfer
     {
+        $moneyValueTransfer = $priceProductTransfer->getMoneyValue();
```

Comment:
* Please compare the pull request with the original one and fix the differences.